flavors/standard-EXASOL-all-java-11/flavor_base/security_scan/packages/apt_get_packages_trivy_deps

```diff
@@ -0,0 +1 @@
+gnupg|2.2.27-3ubuntu2.1
```

Comment:
Do we need this split still?

Reply:
Yes, because first we need to install `gnupg` before we can add the trivy repository to APT (using gnupg) in the Dockerfile

Reply:
ok, gnupg was duplicated in the trivy list itself

Reply:
Fixed it